src/dcap_provider.cpp

```diff
@@ -422,37 +678,6 @@ std::string get_collateral_friendly_name(CollateralTypes collateral_type)
         }
     }
 }
-

```

Comment:
This function has been moved higher in the file so that the region url function can use it